ingestion/src/metadata/workflow/base.py

```diff
@@ -189,6 +191,7 @@ def execute(self) -> None:
 
         # Force resource closing. Required for killing the threading
         finally:
+            logger.debug("%s", self._execution_time_tracker.print_summary())
```

Comment:
Maybe we can add the print here https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/workflow/output_handler.py#L107